test/pylib/rest_client.py

```diff
@@ -196,13 +196,13 @@ async def get_alive_endpoints(self, node_ip: str) -> list:
         assert(type(data) == list)
```

Comment:
I think you should allow `dict[str, Any]` in the APIs, so test authors can conveniently pass dicts such as `{x: True, y: 1}`, and inside `enable_injection` convert the values to strings: `json={key: str(value) for key, value in params.items()}`.